homeassistant/components/airvisual/__init__.py

```diff
@@ -192,7 +192,12 @@ async def async_setup_entry(hass, config_entry):
         cloud_api = CloudAPI(config_entry.data[CONF_API_KEY], session=websession)
 
         async def async_update_data():
-            """Get new data from the API."""
```

Comment:
Why remove the method docstring?

Reply:
Confirm the pull request is rejected with `"""Get new data from the API."""`
![image](https://user-images.githubusercontent.com/59493499/112762857-495b8100-9002-11eb-83b3-65e5132df3dc.png)

Reply:
We shouldn't remove a docstring that's been in place for quite a while because `pylint` is suddenly complaining; we should figure out why.

Try rebasing this branch on `dev` and see if someone pushed anything that might fix this.

Reply:
No way with pylint.
==> My guess, the first string below the fonction name is ok, the second one no
==> That's why currently the string is accepted.
I don't know if you can override.
I will put a code that is granted by all github script

Reply:
Finally I mix both comments.
pylint is too strict or comments shall be one with "#"